db-migration/src/main/resources/db/migration/V2__add_accounts_table.sql

```diff
@@ -1,26 +1,20 @@
-CREATE TABLE roles
-(
+CREATE TABLE roles(
     role_id BIGSERIAL PRIMARY KEY,
```

Comment:
Bigserial is unecessery for role_id field.

Reply:
Done